include/LIEF/ELF/enums.hpp

```diff
@@ -247,7 +247,8 @@ enum class ARCH: size_t  {
   EM_CSR_KALIMBA   = 219, /**< CSR Kalimba architecture family */
   EM_AMDGPU        = 224, /**< AMD GPU architecture */
   EM_RISCV         = 243, /**< RISC-V */
-  EM_BPF           = 247  /**< eBPF Filter */
+  EM_BPF           = 247,  /**< eBPF Filter */
+  EM_LOONGARCH     = 258  /**< Loongson LoongArch */
```

Comment:
Better stay consistent with binutils naming:

```suggestion
  EM_LOONGARCH     = 258  /**< LoongArch */
```